### PR TITLE
Add com.chroniclogic.Zatikon

### PR DESCRIPTION
**Pull request by lukky513**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission

### CODE REVIEW

File: com.chroniclogic.Zatikon.metainfo.xml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+            <li>tons of pieces to choose from</li>
+            <li>army-building metagame</li>
+            <li>magic!</li>
+        </ul>
+        Zatikon offers online and local gameplay, so there's something in it for every kind of player!
+    </p>
+  </description>
+
+  <launchable type="desktop-id">zatikon.desktop</launchable>
+  <content_rating type="oars-1.0">
```
Comment (hfiguiere) [inline com.chroniclogic.Zatikon.metainfo.xml:32]:
```suggestion
  <content_rating type="oars-1.1">
```

File: com.chroniclogic.Zatikon.yml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    type: file
+    dest-filename: zatikon.jar
+    sha256: 1b4dd060bd6cad685dd48f807af59ebd6477bcae944b5ccdf292290fc8fb8115
+  - path: zatikon.sh
+    type: file
+  - path: com.chroniclogic.Zatikon.desktop
+    type: file
+  - path: zatikon128.png
+    type: file
+  - path: zatikon64.png
```
Comment (hfiguiere) [inline com.chroniclogic.Zatikon.yml:44]: you don't need that 64x64 icon.

File: com.chroniclogic.Zatikon.yml [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+finish-args:
+- --env=PATH=/app/jre/bin:/usr/bin:/app/bin
+- --env=JAVA_HOME=/app/jre
+- --device=dri
+- --share=ipc
+- --share=network
+- --socket=x11
+- --socket=pulseaudio
+- --filesystem=xdg-data/zatikon
```
Comment (hfiguiere) [inline com.chroniclogic.Zatikon.yml:17]: why do you need this?

**Comment by hfiguiere** [comment]
bot, build com.chroniclogic.Zatikon

File: com.chroniclogic.Zatikon.metainfo.xml
Comment (hfiguiere) [inline com.chroniclogic.Zatikon.metainfo.xml:1]: this file is never installed.

**Comment by lukky513** [comment]
Fixed all the outstanding issues.

**Comment by barthalion** [comment]
bot, build com.chroniclogic.Zatikon

**Comment by lukky513** [comment]
Added icon to metainfo, hopefully it helps with the current problem.

bot, build com.chroniclogic.Zatikon

**Comment by lukky513** [comment]
Installed from the ref provided by the bot, it appears to be working correctly.

**Comment by lukky513** [comment]
Updated with new binary and removed now-unnecessary permission.

bot, build com.chroniclogic.Zatikon

**Pull request closed without merging**